specs-go/config.go

```diff
@@ -19,6 +20,7 @@ type Spec struct {
 type Device struct {
 	Name string `json:"name"`
 	// Annotations add meta information per device. Note these are CDI-specific and do not affect container metadata.
+	// Added in v0.6.0.
```

Comment:
One thought that I had was whether we wanted to add golang field tags so that we could parse these programatically? This could be done as a follow-up though.